docs/v2-system-addon/data_events.md

```diff
@@ -90,8 +90,53 @@ A user event ping includes some basic metadata (tab id, addon version, etc.) as
   "source": "TOP_SITES",
   "action_position": 2,
   "value": {
-    "card_type": "pinned",
-    "icon_type": ["screenshot_with_icon" | "screenshot" | "tippytop" | "rich_icon" | "no_image"]
+    "card_type": ["pinned" | "search"],
+    "icon_type": ["screenshot_with_icon" | "screenshot" | "tippytop" | "rich_icon" | "no_image"],
+    "search_vendor": ["google" | "amazon" | "baidu" | "yandex"]
```

Comment:
This needs to be added to `Blocking a site` and pinning as well I think. Or would pinning be covered by adding a search shortcut?

Reply:
Yep, this will be added to those two events.